src/chrome/content/rules/CBS.xml

```diff
@@ -14,152 +11,23 @@ Fetch error: http://dw.com.com/ => https://dw.com.com/: (28, 'Connection timed o
 		- ZDNet.xml
 
 
-	om.cnet.com.au/b/ss/zdau-cnet/
-
-
-	CDN buckets:
-
-		- ad.cnet-basic-performance.akadns.net
-
-			- mads.cbs.com
-
-		- wwwimage.cbs.com.edgesuite.net
-
-			- wwwimage.cbsstatic.com
-
-		- www.cbsnews.com.edgesuite.net
-
-		- wwwimage.cbsnews.com.edgesuite.net
-
-			- a1030.g.akamai.net/=/1030/1957/7d/wwwimages.cbsnews.com/	(works!)
-
-		- download-hl.cnet.com.edgesuite.net
-
-			- donwload.cnet.com
-
-		- graphics.fansonly.com.edgesuite.net
-
-			- grfx.cstv.com
-
-		- i.b2b.com.com.edgesuite.net
-
-			- b2b.cbsimg.net
-
-		- i.com.com.edgesuite.net	(maps directly ... but 404s over https.)
-
-			- i.i.cbsi.com
-			- asset\d+.cbsistatic.com
-			- a868.g.akamai.net
-
-				- a248.e.akamai.net/cnwk.1d/
-				- a248.e.akamai.net/=/868/1957/7d/img.com.com/
-
-	cnwk.1d/i/
-	/=/5048/1957/7d/img.com.com/i/
-
-
-	Nonfunctional domains:
```

Comment:
Please double check the `*.cbs.com` hosts and keep existing ones (e.g. `download.cnet.com`)